youtube-gpt-client/src/types/index.ts

```diff
@@ -10,3 +10,8 @@ export interface IconProps {
   color?: string;
 }
 
+export type SummaryData = {
```

Comment:
this data that you will receive will have this structure:

export class VideoResult {
 title: string; // title of the video
 transcription: string; // all the video trasncription
 summary: string; // summary text
 quiz: string // video quiz 
}

We may want to update this later, so far what is done is correct, but it may change when we plug the backend

Reply:
let me change it right now

Reply:
we will need the youtube video url and maybe the id generated from the db